python/Integration/ChimpInsert/config.json

```diff
@@ -0,0 +1,7 @@
+{
+"mailchimp_api" : "",
+"stripe_api" : "",
```

Comment:
Shouldn't this config (as I think it's shared between instances) be in the parent directory? If not, here `stripe_api` would not be necessary?